templates/scripts.tmpl/clean.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#Unofficial strict mode
+set -euo pipefail
+IFS=$'\n\t'
+
+arg1=${1:-}
```

Comment:
This doesn't work if you call it via `npm run`.

Reply:
@willclarktech lerna run doesn't pass the arguments to the child commands. I switched it to lerna exec. Also there was a lerna command for cleaning but just for 'node_modules'. see: https://www.npmjs.com/package/@lerna/clean